lib/protoboeuf/protobuf/uint32value.rb

```diff
@@ -12,9 +12,23 @@ def self.encode(obj)
         obj._encode("").force_encoding(Encoding::ASCII_8BIT)
       end
       # required field readers
-      attr_accessor :value
+
+      attr_reader :value
+
+      def value=(v)
+        unless 0 <= v && v <= 18_446_744_073_709_551_615
```

Comment:
@tenderlove I was admiring your code and found this confusing. Is `uint32` here a 32-bit unsigned value? It looks like this range check is for 64-bit unsigned.